src/main/java/uk/gov/register/views/representations/turtle/RecordTurtleWriter.java

```diff
@@ -31,26 +31,29 @@ public RecordTurtleWriter(javax.inject.Provider<RegisterName> registerNameProvid
     @Override
     protected Model rdfModelFor(RecordView view) {
         Entry entry = view.getEntry();
-        ItemView itemView = view.getItemView();
 
         Model recordModel = ModelFactory.createDefaultModel();
         Model entryModel = new EntryTurtleWriter(registerNameProvider, registerResolver).rdfModelFor(entry, false);
-        Model itemModel = new ItemTurtleWriter(registerNameProvider, registerResolver).rdfModelFor(itemView);
 
         Resource recordResource = recordModel.createResource(recordUri(view.getPrimaryKey()).toString());
         addPropertiesToResource(recordResource, entryModel.getResource(entryUri(Integer.toString(entry.getEntryNumber())).toString()));
-        addPropertiesToResource(recordResource, itemModel.getResource(itemUri(itemView.getItemHash().encode()).toString()));
 
         Map<String, String> prefixes = entryModel.getNsPrefixMap();
-        prefixes.putAll(itemModel.getNsPrefixMap());
+
+        view.getItemViews().forEach(iv -> {
```

Comment:
@michaelabenyohai If the entry for a record has multiple items, and some of those items are duplicates, do we want to show all of the items for the record or just the unique items in the ttl? I suspect the former, as what you would care about when viewing the records is seeing which items belong to it. This is in contrast to when you're viewing an entry, at which time you want t get all of the items, even duplicates.